datasource/tests/tests.py

```diff
@@ -302,3 +306,43 @@ def test_recommend_brands(self):
         self.assertTrue(
             SuggestedAssociation.objects.get(brand=brand2, datasource=usnic5).certainty > 0
         )
+
+
+class FixtureLoadingTestCase(TestCase):
+    def setUp(self):
+        """
+        This method will generate the sample_initial.json file which contains exactly one entry
+        for each module. The size of this file will be very small as compared to the original
+        initial.json
+        """
+
+        file = open("fixtures/initial/initial.json", "r")
+        json_file = json.load(file)
+
+        sample_initial_json = list({data["model"]: data for data in json_file}.values())
+
+        for data in sample_initial_json:
+            data["pk"] = 1
+            for field, field_val in data["fields"].items():
+                if isinstance(field_val, int):
+                    data["fields"][field] = 1
+
+        with open("fixtures/initial/sample_initial.json", "w") as file:
```

Comment:
I realize that this test may take a minute or two to runk, but the purpose of the task is to make sure init.json actually works, always. So you might as well just load initial.json. Otherwise the test isn't actually doing anything at all

Reply:
Let me load then initial.json file instead of dummy json. I will incorporate those changes.